performance-tests/bench_2/worker/configs/simple_action_content_filtered_topics_config.json

```diff
@@ -0,0 +1,111 @@
+{
+  "enable_time": { "sec": -1, "nsec": 0 },
+  "start_time": { "sec": -3, "nsec": 0 },
+  "stop_time": { "sec": -10, "nsec": 0 },
```

Comment:
Let's add an actual run_test.pl to the worker directory that launches the worker with whatever config is specified (e.g. "./worker configs/simple_action_content_filtered_topics_config.json") as if we were going to add it to dcps_test.lst). We'll eventually want to show that the reader received a message to check for correctness, but don't worry about that just yet.  Let's just get it so that your code is exercised. Once we figure out a nice efficient way of checking for success within the worker (we're not quite there yet), we'll add the test to dcps_tests.lst.